static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -58,6 +60,95 @@ function ActionSet({
   const canMarkReviewed =
     anySelected && (allInQuerySelected || selectedIssues.some(issue => !!issue?.inbox));
 
+  // Determine whether to nest "Merge" and "Mark as Reviewed" buttons insde
+  // the dropdown menu based on the current screen size
+  const theme = useTheme();
+  const nestMergeButton = useMedia(`(max-width: ${theme.breakpoints[2]})`);
+  const nestMarkReviewedButton = useMedia(`(max-width: ${theme.breakpoints[1]})`);
+
+  const menuItems: MenuItemProps[] = [
+    ...(nestMergeButton
+      ? [
+          {
+            key: 'merge',
+            label: t('Merge'),
+            onAction: () => {
+              openConfirmModal({
+                bypass: !onShouldConfirm(ConfirmAction.MERGE),
+                onConfirm: () => onMerge(),
+                message: confirm(ConfirmAction.MERGE, false),
+                confirmText: label('merge'),
+              });
+            },
+          },
+        ]
+      : []),
+    ...(nestMarkReviewedButton
+      ? [
+          {
+            key: 'mark-reviewed',
+            label: t('Mark Reviewed'),
+            onAction: () => onUpdate({inbox: false}),
+          },
+        ]
+      : []),
+    {
```

Comment:
Currently, in small viewports, we nest the "Merge" and "Mark as Reviewed" buttons inside the dropdown menu:
<img width="511" alt="Screen Shot 2022-02-10 at 5 45 19 PM" src="https://user-images.githubusercontent.com/44172267/153525647-fa2c4d84-bf73-4fc8-9509-34c667adf0d4.png">

This is done with `hidden-xs`/`hidden-sm`/... classes. The problem with this approach is that in `DropdownMenuV2`, the focus manager doesn't know that these elements should be hidden in larger viewports. The manager will try to pass the focus onto these elements, but visually users will see nothing, no focus ring. 

To fix this, we can programmatically add these menu items to the item list based on the results of media queries. With this, we get the same behavior in small viewports:
<img width="502" alt="Screen Shot 2022-02-10 at 5 52 14 PM" src="https://user-images.githubusercontent.com/44172267/153526157-8f2abfcf-2517-4051-9581-862e38019b84.png">